Sources/PrimerSDK/Classes/Data Models/PrimerConfiguration.swift

```diff
@@ -87,6 +87,12 @@ extension Response.Body {
             return AppState.current.apiConfiguration?.paymentMethods
         }
         
+        var hasSurchargeEnabled: Bool {
+            let pmSurcharge = AppState.current.apiConfiguration?.clientSession?.paymentMethod?.options?.first(where: { $0["surcharge"] as? Int != nil })
+            let cardSurcharge = AppState.current.apiConfiguration?.clientSession?.paymentMethod?.options?.first(where: { /*$0["type"] as? String == "PAYMENT_CARD" &&*/ (($0["networks"] as? [[String: Any]])?.first(where: { $0["surcharge"] as? Int != nil })) != nil  })
```

Comment:
do we need the commented code here?
asking as the constant name is `let cardSurcharge`

Reply:
Nope, we don't